ovirt_imageio/client/_options.py

```diff
@@ -45,6 +45,10 @@ def __repr__(self):
 log_level = Choices("log_level", ("debug", "info", "warning", "error"))
 
 
+def strtobool(str):
```

Comment:
The name describes what this function does, but it is confusing when we use as a type.
Try to find another name that is more consistent with other types names in this module (e.g. log_level).